src/storage/QueryBaseProcessor.inl

```diff
@@ -209,25 +226,37 @@ bool QueryBaseProcessor<REQ, RESP>::checkExp(const Expression* exp) {
         }
         case Expression::kAliasProp:
         case Expression::kEdgeProp: {
-            if (type_ != BoundType::OUT_BOUND) {
-                VLOG(1) << "Only support filter on out bound props";
-                return false;
-            }
-            if (edgeContext_.edgeType_ == -1) {
+            if (edgeContexts_.empty()) {
                 VLOG(1) << "No edge requested!";
                 return false;
             }
+
             auto* edgeExp = static_cast<const AliasPropertyExpression*>(exp);
-            const auto* propName = edgeExp->prop();
-            auto schema = this->schemaMan_->getEdgeSchema(spaceId_, edgeContext_.edgeType_);
+
+            // TODO(simon.liu) we need handle rename.
+            auto edgeStatus = this->schemaMan_->toEdgeType(spaceId_, *edgeExp->alias());
```

Comment:
Could we get the original edgeName from edgeExp?

Reply:
Here we can't get the name before alias(as), so for the time being we don't support the alias of edge. If we want to support alias, the change will be bigger, so I think it would be better to re-open a pr to do this.

Reply:
OK.  Let's open another pr for it.